bin/prepare_project_edition.sh

```diff
@@ -82,7 +82,6 @@ docker exec install_dependencies composer recipes:install ibexa/${PROJECT_EDITIO
 
 # Install Docker stack
 docker exec install_dependencies composer require --dev ibexa/docker:^0.2.x-dev --no-scripts
-docker exec install_dependencies composer sync-recipes ibexa/docker
```

Comment:
This one can be removed, it's included in the call above - the only way this call would make sense would be with `--force` anyway